src/components/app/matching/MatchingCockpit.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
+import { MatchesList } from "./MatchesList";
+import { MatchProps } from "./Match";
+
+interface MatchingCockpitState {
+  matches: Array<MatchProps>
+}
+
+const MATCHES_PLACEHOLDER = {
+  matches: [
+    {
+      matchName: "Tomas",
+      chatStatusIcon: "Chatty",
+      matchIsStarred: true,
+      matchPicture: "awesome"
+    } as MatchProps
+  ]
+} as MatchingCockpitState;
+
+class MatchingCockpit extends React.Component<any, MatchingCockpitState> {
```

Comment:
this could be a pure component as well, as the state will come from the redux store later